src/plugins/borrow-plugins/constants.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export const MAX_FLOAT_PRECISION = 17
```

Comment:
It seems that these constants should be coin-specific, not in a generic constants file that presumably applies to all plugins on all networks. So, I actually liked it the other way, where the constant was in AaveNetwork.js.

So, just grab it from there and skip the whole second commit?

Reply:
Then don't use bignum math? If the result of a calculation is a `double`, biggystring is just burning CPU for a no reason. The only reason to use biggystring is when you need a precise output, down to the Satoshi or Wei or whatever. If you are going to chop off the answer anyhow, just do `parseFloat(string1) / parseFloat(string2)`.